src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -47,14 +47,27 @@
     /**
      * Place a townhall.
      */
-    public static final Achievement achievementBuildingTownhall = new MineColoniesAchievement("townhall", -2, 0, ModBlocks.blockHutTownHall, null)
+    public static final Achievement achievementTownhall = new MineColoniesAchievement("townhall", -2, 0, ModBlocks.blockHutTownHall, null)
+
```

Comment:
Can we get rid of this newline between the object declaration and the registerStat call